azure_pipelines/automation.yml

```diff
@@ -20,6 +20,7 @@ trigger:
   branches:
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->